libs/openFrameworks/types/ofBaseTypes.h

```diff
@@ -848,15 +1681,18 @@ class ofBaseRenderer{
 	/// \param depth The depth of the box.
 	virtual void drawBox( float x, float y, float z, float width, float height, float depth) const;
 
-	/// \brief Draws a cube with the specified size, starting from the specified coordinates.
+	/// \brief Draws a cube using x, y, z, and size.
+	/// coordinates.
 	///
 	/// A cube is a rectangular solid bounded by six square faces of equal size.
```

Comment:
This description was already in the inline documentation. It may belong more in markdown rather than inline.